### PR TITLE
[1.13.0.beta1] Don’t stop iteration on first :class in classNameBindings

### PR DESCRIPTION
**Pull request by mitchlloyd**
Fix for issue #11193.

### CODE REVIEW

File: packages/ember-views/tests/views/view/class_name_bindings_test.js [hunk 1]
```diff
@@ -113,6 +113,16 @@ QUnit.test(":: class name syntax works with an empty true class", function() {
...
 
+QUnit.test("uses all provided static class names (issue #11193)", function() {
+  view = EmberView.create({
+    classNameBindings: [':class-one', ':class-two']
+  });
+
+  run(function() { view.createElement(); });
+
+  equal(view.$().attr('class'), 'ember-view class-one class-two', "both classes are added");
+})
```
Comment (mixonic) [inline packages/ember-views/tests/views/view/class_name_bindings_test.js]: jshint demands a `;`

File: packages/ember-views/lib/system/build-component-template.js [hunk 2]
```diff
@@ -238,7 +238,7 @@ function normalizeClasses(classes, output) {
     // Legacy :class microsyntax for static class names
     if (propName === '') {
       output.push(activeClass);
-      return;
+      continue;
```
Comment (mixonic) [inline packages/ember-views/lib/system/build-component-template.js]: Whooops :-)

File: packages/ember-views/tests/views/view/class_name_bindings_test.js (hunk 1 above)
Comment (mitchlloyd) [inline packages/ember-views/tests/views/view/class_name_bindings_test.js]: JSHint cramping my style again. Fixed.

**Comment by mixonic** [comment]
Thanks man :-)

**Comment by mixonic** [comment]
Drat. I should have been more militant about using `[BUGFIX beta]` in the commit and on this issue @mitchlloyd.

@rwjblue please note this for inclusion in beta!

**Comment by mitchlloyd** [comment]
Well, good to know I'll try to keep that in mind going forward.

**Pull request merged**